src/bindings/python/__init__.py

```diff
@@ -0,0 +1,6 @@
+try:
+    from .python_valhalla import Configure
+except ModuleNotFoundError:
+    from python_valhalla import Configure
```

Comment:
this is only for tests. I couldn't figure out how to tell cmake to build the .so to the `./build/src/bindings/python/valhalla` folder so the tests can also the relative import `from .python_valhalla`